src/models/searchservice/response/directanswer/CustomFieldValueDirectAnswer.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseFieldValueDirectAnswer } from './BaseFieldValueDirectAnswer';
+
+/**
+ * A {@link BaseFieldValueDirectAnswer} with a field type outside of {@link BuiltInFieldType}.
+ *
+ * @public
+ */
+export interface CustomFieldValueDirectAnswer<T = unknown> extends BaseFieldValueDirectAnswer<T>{
```

Comment:
Should this be `UnkownFieldValueDirectAnswer` instead of `CustomFieldValueDirectAnswer`? Since this isn't necessarily a custom field